docs/gaming/monetization.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Monetization in crypto games
```

Comment:
Could we do this one in a separate PR? It's difficult to review among everything else.